deploy/kubernetes/lustre-csi-driver-kind.yaml

```diff
@@ -75,7 +75,7 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: spec.nodeName
-        image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.5.0
+        image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.5.0
```

Comment:
No, don't hand-edit these lustre-csi-driver-*.yaml files.

Instead, edit `deploy/kubernetes/base/plugin.yaml` to fix the csi-node-driver-registrar path.  Then run your "make installer-gen" target to re-build these lustre-csi-driver*.yaml files.

Reply:
Done.